src/WinRPM.jl

```diff
@@ -447,7 +447,13 @@ function do_install(packages::Packages)
     end
 end
 
-const exe7z = iswindows() ? joinpath(BINDIR, "7z.exe") : "7z"
+const exe7z = if try; success(`7z`) catch; false end;
```

Comment:
not a good idea to use any old random version that happens to be on the path, the behavior has been version-specific in ways that have broken this package